sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ReactorReceiver.java

```diff
@@ -38,9 +36,9 @@
     private final ReactorDispatcher dispatcher;
     private final Disposable subscriptions;
     private final AtomicBoolean isDisposed = new AtomicBoolean();
-    private final EmitterProcessor<Message> messagesProcessor;
+    private final Flux<Message> messagesProcessor;
```

Comment:
Is there a Sink that could accomplish what we want here? From my understanding, and correct me if I'm wrong, but this is looking to cache messages until we have a subscriber and once a subscription happens emissions become hot and are only available at emission time and additional subscribers later only receive newly emitted messages.

Reply:
We process messages by transforming them from the handler. There is no programmatic emission.

Reply:
This was an incorrect use of processors before.